src/multiselect_shortcut.js

```diff
@@ -22,7 +22,7 @@ import {MultiselectDraggable} from './multiselect_draggable';
  */
 const registerShortcutDelete = function() {
   const deleteShortcut = {
-    name: Blockly.ShortcutItems.names.DELETE,
+    name: "multiselectDelete",
```

Comment:
It would be good to still use an enum for this: https://github.com/google/blockly/blob/505f28f1a5bf5b8ad65a1b7f333c98671a331996/core/shortcut_items.ts#L23-L34 (Maybe define them in the global.js)

Reply:
Enums do not exist in javascript, so I will make a custom enum type (map) for this. I will write a comment that says that this should be updated should the plugin be updated into Typescript.